feature/experimental/isis/ate_tests/lsp_updates_test/lsp_updates_test.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package rt_2_2_lsp_updates implements RT 2.2 from POPGate vendor device plan
+package rt_2_2_lsp_updates_test
```

Comment:
Normally I remove the numbering from the package name and only keep it as **lsp_updates_test**